### PR TITLE
First task

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Hello @szikszail. Could you please review my completed functions? Thank you

File: module-1/classification.js [hunk 1]
```diff
@@ -23,8 +24,24 @@ function grade(score) {
      * Also take into consideration the documentation of the function!
      */
     // PLACE YOUR CODE BETWEEN THIS...
-
+    if(score < 1 && score > 100) {
+        return 0;
+    }
+    if (0 <= score && score <= 59) {
+        gradeOfStudent = 1;
+    } else if (60 <= score && score <= 69) {
```
Comment (szikszail) [inline module-1/classification.js]: Do you need to check whether it is bigger or equal than 60?

File: module-1/classification.js [hunk 2]
```diff
@@ -23,8 +24,24 @@ function grade(score) {
...
+        gradeOfStudent = 2;
+    } else if (70 <= score && score <= 79) {
```
Comment (szikszail) [inline module-1/classification.js]: Do you need to check whether it is bigger or equal than 70?

File: module-1/classification.js [hunk 3]
```diff
@@ -23,8 +24,24 @@ function grade(score) {
...
+        gradeOfStudent = 3;
+    } else if (80 <= score && score <= 89) {
```
Comment (szikszail) [inline module-1/classification.js]: Do you need to check whether it is...? I think you know what I mean :D

File: module-1/euclidean.js [hunk 4]
```diff
@@ -18,8 +18,15 @@ function euclidean(a, b) {
      * Also take into consideration the documentation of the function!
      */
     // PLACE YOUR CODE BETWEEN THIS...
-
-
+    if (a < 0 || b < 0) return 0;
+    let min = a < b ? a : b;
+    let max = a > b ? a : b;
+    var euclideNumbers = [];
```
Comment (szikszail) [inline module-1/euclidean.js]: Good approach, but why do you need to store all "euclide numbers"?

File: module-1/fibonacci.js [hunk 5]
```diff
@@ -17,7 +17,14 @@ function fibonacci(n) {
      * Also take into consideration the documentation of the function!
      */
     // PLACE YOUR CODE BETWEEN THIS...
-
+    let secondVal;
+    if (n < 0) return 0;
+    let arr = [0, 1];
+    for (var i = 1; i <= 50; i++) {
```
Comment (szikszail) [inline module-1/fibonacci.js]: Determining all the first 50 items of the series seems unnecessary, especially if you only need to return e.g. the 2. :) Of course if we would expect that this function will be used tones of times, but then this should be done separately, not every time this function is called.

**Review by szikszail — CHANGES_REQUESTED** [review]
Please check my comments. I'd be happy to review your next solutions.

File: module-1/classification.js (hunk 1 above)
Comment (ghost) [reply module-1/classification.js]: Yes I see. I can check just for score <= 69. Does it work?

Comment (ghost) [reply module-1/classification.js]: Changed

File: module-1/classification.js (hunk 2 above)
Comment (ghost) [reply module-1/classification.js]: Changed it

File: module-1/classification.js (hunk 3 above)
Comment (ghost) [reply module-1/classification.js]: Changed it

File: module-1/euclidean.js (hunk 4 above)
Comment (ghost) [reply module-1/euclidean.js]: I deleted this array

File: module-1/fibonacci.js (hunk 5 above)
Comment (ghost) [reply module-1/fibonacci.js]: Yes I see. We can just put here method parameters.

File: module-1/classification.js [hunk 6]
```diff
@@ -23,8 +24,24 @@ function grade(score) {
...
+        gradeOfStudent = 1;
+    } else if (score <= 69) {
+        gradeOfStudent = 2;
+    } else if (score <= 79) {
+        gradeOfStudent = 3;
+    } else if (score <= 89) {
+        gradeOfStudent = 4;
+    } else if (score <= 100) {
+        gradeOfStudent = 5;
+    } else {
```
Comment (szikszail) [inline module-1/classification.js]: I don't think that this path will ever be executed.

File: module-1/fibonacci.js [hunk 7]
```diff
@@ -17,7 +17,14 @@ function fibonacci(n) {
      * Also take into consideration the documentation of the function!
      */
     // PLACE YOUR CODE BETWEEN THIS...
-
+    let secondVal;
+    if (n < 0) return 0;
+    let arr = [0, 1];
+    for (var i = 1; i <= n; i++) {
```
Comment (szikszail) [inline module-1/fibonacci.js]: Are you sure that `i <= n` is correct here? I think with this you do an unnecessary execution of the loop.

**Review by szikszail — CHANGES_REQUESTED** [review]

File: module-1/classification.js (hunk 6 above)
Comment (ghost) [reply module-1/classification.js]: Yes. It can be removed.

File: module-1/fibonacci.js (hunk 7 above)
Comment (ghost) [reply module-1/fibonacci.js]: I just limited it. I don't know to make it in the different way.

**Comment by ghost** [comment]
I closed it because of my incorrect changes and created separate mr.

**Pull request closed without merging**